src/qtiItem/core/Element.js

```diff
@@ -448,12 +448,12 @@ Element.issetElement = function(serial) {
  * @param {String} serial - the element serial
  * @returns {Boolean} true if unset
  */
-Element.unsetElement = function(serial) {
+Element.unsetElement = function (serial) {
     var element = Element.getElementBySerial(serial);
 
     if (element) {
         var composingElements = element.getComposingElements();
-        _.each(composingElements, function(elt) {
+        _.each(composingElements, function (elt) {
```

Comment:
https://lodash.com/docs/4.17.15#forEach
```suggestion
        _.forEach(composingElements, function (elt) {
```